cmd/virt-handler/virt-handler.go

```diff
@@ -674,6 +677,9 @@ func (app *virtHandlerApp) AddFlags() {
 
 	flag.IntVar(&app.gracefulShutdownSeconds, "graceful-shutdown-seconds", defaultGracefulShutdownSeconds,
 		"The number of seconds to wait for existing migration connections to close before shutting down virt-handler.")
+
+	flag.BoolVar(&app.disableNodeLabeller, "disable-node-labeller", false,
+		"Disable Node Labeller controller to prevent the node labelling functionality.")
```

Comment:
I think it's preferred to have an enable flag rather than a disable one.
IOW, I'd suggest:
```suggestion
	flag.BoolVar(&app.disableNodeLabeller, "enable-node-labeller", true,
		"Enable Node Labeller controller.")
```